tests/test_enterprise/api_client/test_discovery.py

```diff
@@ -24,26 +24,20 @@ class TestCourseCatalogApiInitialization(unittest.TestCase):
     @mock.patch('enterprise.api_client.discovery.CatalogIntegration')
     @mock.patch('enterprise.api_client.discovery.get_edx_api_data')
     def test_raise_error_missing_course_discovery_api(self, *args):  # pylint: disable=unused-argument
-        message = 'To get a Catalog API client, this package must be installed in an Open edX environment.'
-        with raises(NotConnectedToOpenEdX) as excinfo:
+        with self.assertRaises(NotConnectedToOpenEdX):
             CourseCatalogApiClient(mock.Mock(spec=User))
-        assert message == str(excinfo.value)
```

Comment:
I've a pet peeve about this: there's no reason to check for the exact error message unless you're testing (for some reason) the exact error message you're getting. If all you're worried about *is that the exception's raised*, then only test for that. In this test (and honestly others in the codebase), what's being tested is that the error is raised (see the function name). Keeping the error string in there causes a potential maintenance headache.